Traceback (most recent call last):
  File "/Users/emily/anaconda3/envs/MACS497/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/Users/emily/anaconda3/envs/MACS497/lib/python3.10/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/emily/anaconda3/envs/MACS497/lib/python3.10/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/emily/anaconda3/envs/MACS497/lib/python3.10/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/Users/emily/anaconda3/envs/MACS497/lib/python3.10/asyncio/base_events.py", line 646, in run_until_complete
    return future.result()
  File "/Users/emily/anaconda3/envs/MACS497/lib/python3.10/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/Users/emily/anaconda3/envs/MACS497/lib/python3.10/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/emily/anaconda3/envs/MACS497/lib/python3.10/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# binary operations act differently on different types of objects
c = 'World'
print(b + c)
print(a + 2)
print(a + b)
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mTypeError[0m                                 Traceback (most recent call last)
Input [0;32mIn [8][0m, in [0;36m<cell line: 5>[0;34m()[0m
[1;32m      3[0m [38;5;28mprint[39m(b [38;5;241m+[39m c)
[1;32m      4[0m [38;5;28mprint[39m(a [38;5;241m+[39m [38;5;241m2[39m)
[0;32m----> 5[0m [38;5;28mprint[39m([43ma[49m[43m [49m[38;5;241;43m+[39;49m[43m [49m[43mb[49m)

[0;31mTypeError[0m: unsupported operand type(s) for +: 'int' and 'str'
TypeError: unsupported operand type(s) for +: 'int' and 'str'

